scripts/inference.py

```diff
@@ -184,6 +193,22 @@ def pad_prompt(prompt, pad_len, pad_token="<unk>"):
 
 ids = prompt1.unsqueeze(0)
 
+if args.compile:
+    print("compiling model")
+    # compiling can make first inference pass slow
+    model = torch.compile(model, mode=args.compile_mode)
```

Comment:
if compile and export are both true, looks like this model is overwritten, we may want to check for export then in elif, do a compile. Also may want to add a comment for compile that if export is set, compile will be ignored.